examples/nlp/neural_text_normalization/neural_text_normalization_train.py

```diff
@@ -0,0 +1,35 @@
+from omegaconf import DictConfig, OmegaConf
+from utils import TAGGER_MODEL, DECODER_MODEL, initialize_model_and_trainer
+
+from nemo.utils import logging
+from nemo.core.config import hydra_runner
+from nemo.utils.exp_manager import exp_manager
+
```

Comment:
could you add more information on task, how to use the arguments etc? following the example of text_classification.py or token classification?